s3bench.py

```diff
@@ -210,10 +216,18 @@ def list_random_objects(self):
             # generates new object's name
             object_name_given = object_analyzer.generate_object_name()
 
+            # add prefix to object name according its size
```

Comment:
1. We shouldn't use this line, mainly because we want the `generate_object_name` function to return the proper path for the object.